### PR TITLE
JP - Allows multi-level deep packages when defining libraries

### PR DESCRIPTION
**Pull request by juanpedromoreno**
Fixes https://github.com/scala-exercises/scala-exercises/issues/554

Please, @raulraja @dialelo could you review? Thanks.

### CODE REVIEW

**Comment by dialelo** [comment]
LGTM, great job!

**Comment by andyscott** [comment]
Could you add a scripted test for this new functionality? That way, if we revamp any of the compiler code, we can ensure compatibility?

**Comment by juanpedromoreno** [comment]
Sure, will do.

**Comment by juanpedromoreno** [comment]
@andyscott Done, please, take a look. Thanks.

File: sbt-exercise/src/sbt-test/sbt-exercise/basic/build.sbt [hunk 1]
```diff
@@ -14,8 +14,22 @@ lazy val content = (project in file("content"))
     )
 )
 
+lazy val contentinpackages = (project in file("contentinpackages"))
```
Comment (andyscott) [inline sbt-exercise/src/sbt-test/sbt-exercise/basic/build.sbt]: Minor nitpick: maybe make `contentinpackages` camel case?

**Comment by andyscott** [comment]
@juanpedromoreno Other than the minor nitpick, looks great!

**Pull request merged**